vllm/engine/llm_engine.py

```diff
@@ -1472,11 +1586,10 @@ def _get_stats(
                 # the same metadata more than once per request, we standardize
                 # on logging request level information for finished requests,
                 # which can only happen once.
-                if seq_group.is_finished():
+                if (seq_group.is_finished()):
```

Comment:
Seems not necessary to change this line?

Reply:
fixed